src/Microsoft.Azure.ServiceBus/Amqp/AmqpResponseMessage.cs

```diff
@@ -79,5 +79,12 @@ public IEnumerable<TValue> GetListValue<TValue>(MapKey key)
 
             return list.Cast<TValue>();
         }
+
+        public AmqpSymbol GetResponseErrorCondition()
```

Comment:
My bad. I removed another function which was referencing this and didn't clean this up.